src/drive/mobile/modules/mediaBackup/duck/tmp.js

```diff
@@ -0,0 +1,63 @@
+/* global cozy */
+/* 
+This file is TEMPORARY. It covers a stack bug and we need to synchronize again some files
```

Comment:
Maybe we can give the file or more explicit name than `tmp`? `checkCorruptedFiles` is actually pretty good!